src/main/java/edu/ucsb/cs156/dining/controllers/ReviewController.java

```diff
@@ -173,14 +160,13 @@ public Review editReview(@Parameter Long id, @RequestBody @Valid EditedReview in
 
         if (incoming.getReviewerComments() != null &&!incoming.getReviewerComments().trim().isEmpty()) {
             oldReview.setReviewerComments(incoming.getReviewerComments());
-            oldReview.setStatus(ModerationStatus.AWAITING_REVIEW);
         }else{
             oldReview.setReviewerComments(null);
-            oldReview.setStatus(ModerationStatus.APPROVED);
         }
 
         oldReview.setDateItemServed(incoming.getDateItemServed());
 
+        oldReview.setStatus(ModerationStatus.AWAITING_REVIEW);
```

Comment:
Should we not do this ONLY if the updated reviewer comments are non-empty and have changed?

Shouldn't we keep the old status if the reviewer comments are unchanged, or set it to APPROVED if the new reviewer comments are empty?